packages/jest-preset-default/jest-preset.json

```diff
@@ -0,0 +1,27 @@
+{
+	"coverageDirectory": "coverage",
+	"coveragePathIgnorePatterns": [
+		"<rootDir>/.*/build.*"
+	],
+	"moduleNameMapper": {
+		"\\.(scss|css)$": "<rootDir>/packages/jest-preset-default/scripts/style-mock.js"
```

Comment:
All occurences of `<rootDir>/packages/jest-preset-default` will have to be replaced with:
`<rootDir>/node_modules/@wordpress/jest-preset-default`